[2025-10-02T05:52:45.239+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-02T05:52:45.246+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T05:52:43.264598+00:00 [queued]>
[2025-10-02T05:52:45.249+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T05:52:43.264598+00:00 [queued]>
[2025-10-02T05:52:45.250+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2025-10-02T05:52:45.258+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): download_raw_from_s3> on 2025-10-02 05:52:43.264598+00:00
[2025-10-02T05:52:45.262+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=451) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-10-02T05:52:45.263+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'sentiment140_end2end_single', 'download_raw_from_s3', 'manual__2025-10-02T05:52:43.264598+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/edaproject.py', '--cfg-path', '/tmp/tmp9ncj5i2j']
[2025-10-02T05:52:45.268+0000] {standard_task_runner.py:91} INFO - Job 32: Subtask download_raw_from_s3
[2025-10-02T05:52:45.267+0000] {standard_task_runner.py:63} INFO - Started process 453 to run task
[2025-10-02T05:52:45.292+0000] {task_command.py:426} INFO - Running <TaskInstance: sentiment140_end2end_single.download_raw_from_s3 manual__2025-10-02T05:52:43.264598+00:00 [running]> on host 590373856cb4
[2025-10-02T05:52:45.329+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='sentiment140_end2end_single' AIRFLOW_CTX_TASK_ID='download_raw_from_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-10-02T05:52:43.264598+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-10-02T05:52:43.264598+00:00'
[2025-10-02T05:52:45.331+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-02T05:52:45.335+0000] {logging_mixin.py:188} INFO - [download_raw_from_s3] Using CSV_S3_URI=s3://298a1/sentiment140/***/main-data.csv
[2025-10-02T05:52:45.339+0000] {base.py:84} INFO - Using connection ID 'aws_default' for task execution.
[2025-10-02T05:52:45.339+0000] {connection_wrapper.py:384} INFO - AWS Connection (conn_id='aws_default', conn_type='aws') credentials retrieved from login and password.
[2025-10-02T05:52:52.095+0000] {logging_mixin.py:188} INFO - [download_raw_from_s3] Downloaded to /tmp/MAUD_train_raw.csv (238803811 bytes)
[2025-10-02T05:52:52.136+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-10-02T05:52:52.144+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-02T05:52:52.151+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=sentiment140_end2end_single, task_id=download_raw_from_s3, run_id=manual__2025-10-02T05:52:43.264598+00:00, execution_date=20251002T055243, start_date=20251002T055245, end_date=20251002T055252
[2025-10-02T05:52:52.186+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-02T05:52:52.226+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-10-02T05:52:52.227+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
